help/en/package/jmri/jmrit/logixng/ConditionalNGDebugger.shtml

```diff
@@ -22,7 +22,9 @@
       <p>To use the debugger, it must be enabled. In <strong>Preferences &rArr; LogixNG</strong>,
       the option <strong>Install debugger</strong> must be enabled.</p>
 
-
+      <p>The debugger supports two modes.  The first one is an IDE style debugger which supports
```

Comment:
This PR looks good. Thanks.

My only concern is that "mode" usually means one of several options there you can only choose one option at the time. For example a power supply that gives AC or DC, but not both at the same time.

In this case, you can both use the IDE style debugger and at the same time use the debugger logger statements. You don't need to choose one or another.

But I don't know if this is an issue for the users. If you want to keep the text as is, it's fine for me.